tools/build.sh

```diff
@@ -20,7 +20,7 @@ while getopts ":l:m:d:p:" o; do
       ;;
     p)
       pwm=${OPTARG}
-      ((pwm == 24 || pwm == 48 || pwm == 96)) || usage
+      ((pwm == 24 || pwm == 48 || pwm == 96 || pwm == DYNAMIC)) || usage
```

Comment:
I think I mentioned this in one of the other PRs already, but we don't really need this anymore. By default we should compile only the 24kHz version, the rest the user can setup in the configurator anyway.

Reply:
I think we could move to a folder named Modules and remove Bluejay prefix. Please, let me know if you prefer a different name.

Reply:
Sounds good to me!

Reply:
done

Reply:
done